pyduckling/__init__.py

```diff
@@ -19,4 +21,9 @@ def parse_time(text, lang='EN', date=0):
     :returns: The duckling json representation of all found time
               expressions.
     """
-    return json.loads(parse(text, lang, date))
+    try:
+        type(date) != int
```

Comment:
This seems a little weird - why not use

```
if type(date) != int:
    ...
 else:
    ...
```
instead of `try...except...`?

Reply:
Other than that - thanks a lot!

Reply:
I did that because I made the assumption that the date type would be much more a string than an int, but it probably doesn't matter. :-)